contributors/design-proposals/storage/dynamic_volume_limit.md

```diff
@@ -12,20 +12,21 @@ limits depending on type of node.
 
 ### Prerequisite
 
-* This feature will be protected by an alpha feature gate, so as API and CLI changes needed for it. We are planning to call
+* 1.11 This feature will be protected by an alpha feature gate, so as API and CLI changes needed for it. We are planning to call
   the feature `AttachVolumeLimit`.
+* 1.12 This feature will be behind a beta feature gate and enabled by default.
 
 ### API Changes
 
 There is no API change needed for this feature. However existing `node.Status.Capacity` and `node.Status.Allocatable` will
 be extended to cover volume limits available on the node too.
 
-The key name that will store volume will be start with prefix `storage-attach-limits-`. The volume limit key will respect
+The key name that will store volume will be start with prefix `attachable-volumes-`. The volume limit key will respect
 format restrictions applied to Kubernetes Resource names. Volume limit key for existing plugins might look like:
 
 
-* `storage-attach-limits-aws-ebs`
-* `storage-attach-limits-gce-pd`
+* `attachable-volumes-aws-ebs`
```

Comment:
Given that we also want to add a max capacity limit, does "attachable-volumes" still make sense, or do we want to clarify that it is a count?

Reply:
I was thinking to use separate resource name for max attachable capacity. Something like - `attachable-capacity-xxx` and keep `attachable-volumes-xx` for representing count. I think capacity vs volumes word is "good enough" indication.